daemon/graphdriver/aufs/aufs.go

```diff
@@ -461,8 +460,7 @@ func (a *Driver) DiffGetter(id string) (graphdriver.FileGetCloser, error) {
 
 func (a *Driver) applyDiff(id string, diff io.Reader) error {
 	return chrootarchive.UntarUncompressed(diff, path.Join(a.rootPath(), "diff", id), &archive.TarOptions{
-		UIDMaps: a.uidMaps,
-		GIDMaps: a.gidMaps,
+		IDMap: idtools.NewIDMappingsFromMaps(a.uidMaps, a.gidMaps),
```

Comment:
Wondering why you're introducing the deprecated utility in these intermediate commits, before replacing them later, or was that just to make it easier to find back which locations had to be updated? https://github.com/moby/moby/blob/7b9275c0da707b030e62c96b679a976f31f929d3/pkg/idtools/idtools.go#L119